tests/basic/storage_pool_test.go

```diff
@@ -5794,3 +5794,226 @@ func addNewPools(n node.Node, numPools int) error {
+
+		log.InfoD("Bring Node out of Maintenance Mode")
+		ExitFromMaintenanceMode(*nodeDetail)
+
+		// Do Pool Expand on the Node
+		stepLog = fmt.Sprintf("Expanding pool on node [%s] and pool UUID: [%s] using auto", nodeDetail.Name, poolUUID)
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			poolToBeResized, err := GetStoragePoolByUUID(poolUUID)
+			log.FailOnError(err, "Failed to get pool using UUID [%s]", poolUUID)
+			expectedSize := poolToBeResized.TotalSize * 2 / units.GiB
+
+			isjournal, err := isJournalEnabled()
+			log.FailOnError(err, "Failed to check if Journal enabled")
+
+			log.InfoD("Current Size of the pool [%s] is [%d]", poolUUID, poolToBeResized.TotalSize/units.GiB)
+			err = Inst().V.ExpandPool(poolUUID, api.SdkStoragePool_RESIZE_TYPE_AUTO, expectedSize)
+			dash.VerifyFatal(err, nil, "Pool expansion init successful?")
+
+			resizeErr := waitForPoolToBeResized(expectedSize, poolUUID, isjournal)
+			dash.VerifyFatal(resizeErr, nil, fmt.Sprintf("Verify pool [%s] on node [%s] expansion using auto", poolUUID, nodeDetail.Name))
+		})
+
+		// Validate if PoolIO Priority is not changed after pool Expansion
+		ioPriorityAfter, err := Inst().V.GetPoolLabelValue(poolUUID, "iopriority")
+		log.FailOnError(err, "Failed to get IO Priority for Pool with UUID [%v]", poolUUID)
+
+		log.InfoD(fmt.Sprintf("Priority Before [%s] and Priority after Pool Expansion [%s]", ioPriorityBefore, ioPriorityAfter))
+		dash.VerifyFatal(ioPriorityAfter == setIOPriority, true, "IO Priority mismatch after pool expansion")
+
+	})
+
+	JustAfterEach(func() {
+		defer EndTorpedoTest()
+		log.InfoD("Exit from Maintenance mode if Pool is still in Maintenance")
+		ExitNodesFromMaintenanceMode()
+		AfterEachTest(contexts, testrailID, runID)
+	})
+})
+
+var _ = Describe("{VerifyPoolDeleteInvalidPoolID}", func() {
+	var testrailID = 79487
+	// Testrail Description : Verify deletion of invalid pool ids
+	// Testrail Corresponds : https://portworx.testrail.net/index.php?/cases/view/55349
+	var runID int
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("VerifyPoolDeleteInvalidPoolID",
+			"Verify deletion of invalid pool ids",
+			nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+
+	var contexts []*scheduler.Context
+	stepLog := "Verify deletion of invalid pool ids"
+	It(stepLog, func() {
+		log.InfoD(stepLog)
+
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("deleteinvalidpoolid-%d", i))...)
+		}
+		ValidateApplications(contexts)
+		defer appsValidateAndDestroy(contexts)
+
+		// Get the Pool UUID on which IO is running
+		poolUUID, err := GetPoolIDWithIOs()
+		log.InfoD("Pool UUID on which IO is running [%s]", poolUUID)
+		log.FailOnError(err, "Failed to get pool using UUID [%v]", poolUUID)
```

Comment:
Same here, If `GetPoolIDWithIOs()` fails, it won't return `poolUUID`, and always `log.FailOnError()` first before doing `log.InfoD()`

Reply:
fixed